edit/settings.js

```diff
@@ -27,7 +27,7 @@ function StyleSettings(editor) {
 
   function update(newStyle, reason) {
     if (!newStyle.id) return;
-    if (reason === 'editSave' || reason === 'config') return;
+    if (reason === 'editSave') return;
```

Comment:
Why?

Reply:
We have to refresh the setting page when style setting changes outside of the setting page. For example:

1. Install a style.
2. Open style setting page.
3. Open the popup, toggle exclusions.

You will see that the change to exclusions is also reflected on style setting page.

Should we add some comment?

Reply:
Update the page if it's not modified or show our standard question styleUpdateDiscardChanges?

Reply:
Note that setting page doesn't bind to the editor and the dirty system. When a setting is changed, it calls the API `API.styles.config` immediately. You can verify this by adding `*` to inclusions. The style will be applied to the page when the textarea loses focus.

Unlike editor, setting page doesn't store a "change", so there is nothing can be discarded.

----

For the editor, there is a similar situation: toggling a style. When the editor is dirty, toggling a style in the popup will also toggle the `Enabled` checkbox on the page, which modifies the dirty state. When the editor is not dirty, the checkbox is toggled without modifying the dirty state.

Reply:
Ah, I misunderstood the change. Now I see it adds the support, not removes it.